doc/API.md

```diff
@@ -711,253 +711,190 @@ The input byte array should be 80 bytes that form the solved block header. *Unli
 Renter
 ------
 
-Queries:
-
-* /renter/allowance          [GET]
-* /renter/allowance          [POST]
-* /renter/downloads          [GET]
-* /renter/files              [GET]
-* /renter/load               [POST]
-* /renter/loadascii          [POST]
-* /renter/share              [GET]
-* /renter/shareascii         [GET]
-* /renter/delete/{siapath}   [POST]
-* /renter/download/{siapath} [GET]
-* /renter/rename/{siapath}   [POST]
-* /renter/upload/{siapath}   [POST]
+| Route                                                         | HTTP verb |
+| ------------------------------------------------------------- | --------- |
+| [/renter](#renter-get)                                        | GET       |
+| [/renter](#renter-post)                                       | POST      |
+| [/renter/contracts](#rentercontracts-get)                     | GET       |
+| [/renter/downloads](#renterdownloads-get)                     | GET       |
+| [/renter/files](#renterfiles-get)                             | GET       |
+| [/renter/delete/___*siapath___](#renterdeletesiapath-post)    | POST      |
+| [/renter/download/___*siapath___](#renterdownloadsiapath-get) | GET       |
+| [/renter/rename/___*siapath___](#renterrenamesiapath-post)    | POST      |
+| [/renter/upload/___*siapath___](#renteruploadsiapath-post)    | POST      |
```

Comment:
If the group gets too large it makes sense to break it apart into two smaller groups. Each group should be sorted alphabetically, of course.